Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -315,6 +334,14 @@ protected virtual void Update()
             }
 
             lastState = StateManager.CurrentState();
+            
+            if (requiresFocusValueCheck != RequiresFocus || isGlobalValueCheck != IsGlobal)
```

Comment:
I'm now realizing that RequiresFocus was documented, but I was confused because I didn't see it in our [API docs](https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.UI.Interactable.html). I think if you use a format like this then the docs will populate, right @thalbern?

```
        /// <summary>
        /// Does the voice command require this to have focus?
        /// </summary>
        public bool RequiresFocus = true;
```

Reply:
I updated a ton of single line comments to be summary comments. Hopefully this will help with feature discovery.

Reply:
Thanks!